share/completions/eselect.fish

```diff
@@ -33,12 +34,19 @@ function __fish_complete_eselect_action_options
     __fish_eselect_cmd $cmdl[2..-1] | sed -n -re $findregexp | __fish_sgrep '^    --' | sed -re $parseregexp
 end
 
+function __fish_complete_eselect_php_actions
+    set -l sedregexp 's/^  .([0-9]+).   ([A-Za-z0-9\.]+).?+/\1\t\2/'
+
+    if test (__fish_print_cmd_args_without_options)[3] = 'set'
+        eselect php list (__fish_print_cmd_args_without_options)[-1] | sed -r $sedregexp
```

Comment:
>I didn't manage to build fish from git, it just breaks.

What breaks, what OS are you on? You should be doing `autoreconf; ./configure; make; make install` - I think the dependencies only changed in version (though you can use a system-provided pcre2, we'll default to a bundled version).

>The one with the star is the selected version.

That might be nice to add to the description, though it might need some more advanced regex-foo.

>I just put a commit in with sed since I can't get the git version to behave so I can't test it.

It's not critical, we can replace it later.

I'll now try to get a gentoo install working (though I find the install process quite tedious and have gotten a few rather weird errors) to confirm this works.